apps/webapp/src/pages/LeadSubmissionsPage.tsx

```
@@ -0,0 +1,136 @@
+    ];
+    const hasValue = (v: any) => {
+      if (v == null) return false;
+      const s = String(v);
+      return s.trim().length > 0;
+    };
+    // Only keep columns that have at least one non-empty value across rows
+    return defs.filter((col) => rows.some((r) => hasValue(col.get(r))));
+  }, [rows]);
+
+  const csv = useMemo(() => {
+    const escape = (v: any) => {
```

Biome (2.1.2)

[error] Do not shadow the global "escape" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)